hi-ml-cpath/src/health_cpath/models/transforms.py

```diff
@@ -26,16 +26,33 @@ def load_pil_image(image_path: PathOrString) -> PIL.Image.Image:
     return image
 
 
-def load_image_as_tensor(image_path: PathOrString) -> torch.Tensor:
-    """Load an image as a tensor from the given path"""
-    pil_image = load_pil_image(image_path)
-    return to_tensor(pil_image)
+def load_image_as_tensor(image_path: PathOrString, scale_intensity: bool = True) -> torch.Tensor:
+    """Load an image as a tensor from the given path
+
+    :param image_path: path to the image
+    :param scale_intensity: if True, use `to_tensor` from torchvision which scales the image pixel intensities to
```

Comment:
In the function description here and in line 44ff, it would be good to emphasize that the returned tensor is _always_ in CHW format. At present, it looks as if this might be the case only for `scale=False`.

Reply:
Good point, will do!